pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,18 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// HandlerWithName holds information for handler, infoResp.Name
```

Comment:
what is an `infoResp.Name`? It's not clear from the local context what that is referring to. Could you augment the docstring to be more clear as to what this is actually storing. a "handler" and "infoResp" are very generic